targets/targets.json

```diff
@@ -1192,7 +1192,7 @@
         },
         "extra_labels_add": ["STM32F4", "STM32F429", "STM32F429ZI", "STM32F429xx", "STM32F429xI"],
         "macros_add": ["USB_STM_HAL", "USBHOST_OTHER"],
-        "device_has_add": ["ANALOGOUT", "CAN", "LOWPOWERTIMER", "SERIAL_ASYNCH", "SERIAL_FC", "TRNG", "FLASH"],
+        "device_has_add": ["RTC", "ANALOGOUT", "CAN", "LOWPOWERTIMER", "SERIAL_ASYNCH", "SERIAL_FC", "TRNG", "FLASH"],
```

Comment:
RTC is already inherited from FAMILY_STM32

Reply:
RTC has been disabled on the feature branch for all targets (https://github.com/ARMmbed/mbed-os/commit/bdcb3d239e92508d783fef8ae0f7074141f5b2fb). This PR enables RTC only for NUCLEO-F429ZI.

Reply:
So, please, enable RTC back in FAMILY_STM32 level.
I am going to provide test report for all STM32 targets.

Thx